guide/sample-webots-applications.md

```diff
@@ -1,11 +1,11 @@
 # Sample Webots Applications
 
 This chapter gives an overview of sample worlds provided with the Webots
-package. The examples world can be tried easily; the ".wbt" files are located in
+package. The example worlds can be tested easily; the ".wbt" files are located in
 various "worlds" directories of the "WEBOTS\_HOME/projects" directory and can be
 directly opened from Webots using the `Open Sample World` item in `File` menu.
 The controller code is located in the corresponding "controllers" directory.
-This chapter provides each example a with short abstract only. More detailed
+This chapter provides a only short abstract for each example. More detailed
```

Comment:
It looks strange to me: I would have written "only a short" instead of "a only short"...